core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -326,7 +326,7 @@ trait WhiskWebActionsApi
     private val defaultCorsResponse = List(
         `Access-Control-Allow-Origin`(AllOrigins),
         `Access-Control-Allow-Methods`(OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH),
-        `Access-Control-Allow-Headers`(`Authorization`.name, `Content-Type`.name))
+        `Access-Control-Allow-Headers`(`Authorization`.name, `Content-Type`.name, `User-Agent`.name))
```

Comment:
But the controller does nothing with these user agents for POST. And accepting it here for web actions is just for preflight - I suppose it's fine (but for the cli this has no value there). If you want to keep it I think we need a doc update.